### ISSUE TITLE
Version 0.1.3 Release

### ISSUE THREAD

**Issue by huytd** [huytd/pomoday-v2#33]
This is the tracker ticket for the progress of `v0.1.3` release. The finished items will be deployed right away on production once they're done.

Release note will come with a usage guide for each finished feature once the release is finished.

This version focus on task editing and organizing experience:

- [x] Feature: Delete a tag
- [x] Feature: Toggle task visibilty by status
- [x] Feature: Delete task by status
- [x] Enhancement: Make each split scroll separately
- [x] Feature: Archive task (hide them but not delete them from the history)
- [x] Feature: Display tag progress
- [x] Feature: Command input enhancement
- [x] Customizable UI

**Comment by hgoldstein** [comment]
Two questions about archived tasks:
* I'm assuming `unarchive` would also be useful here.
* Auto-archiving completed tasks seems desirable as well, if not out of scope for this task.

**Comment by huytd** [comment]
@hgoldstein yes there will be the ability to archive/unarchive/show archived items.

For auto archive, that's a good idea, depends on how easy it is, I'll include it in v0.1.3 (should be easy).

**Comment by huytd** [comment]
A quick overview of customizable UI

![](https://user-images.githubusercontent.com/613943/69000561-716f9780-0886-11ea-8f67-c3ac7d125d19.png)
